ci/manifest.yml.tmpl

```diff
@@ -0,0 +1,28 @@
+---
+applications:
+  - name: test-rp-msa-$ENV
+    routes:
+      - route: test-rp-msa-$ENV.cloudapps.digital
+      - route: test-rp-msa-$ENV.apps.internal
+    memory: 1G
+    buildpack: java_buildpack
+    env:
+      JBP_CONFIG_OPEN_JDK_JRE: '{ jre: { version: 11.+ } }'
+      CONFIG_FILE: /app/msa/test-rp-msa.yml
+      LOG_LEVEL: INFO
+      TEST_RP_MSA_URL: https://test-rp-msa-$ENV.cloudapps.digital/matching-service/POST
+      TEST_RP_ROUTE: http://test-rp-$ENV.apps.internal:8080
+      METADATA_URL: $METADATA_URL
+      SIGNIN_DOMAIN: $SIGNIN_DOMAIN
+      METADATA_ENTITY_ID: https://signin.service.gov.uk
+      TRUSTSTORE_PATH: /app/msa/truststores/$TRUSTSTORE_NAME
+      TRUSTSTORE_PASSWORD: $TRUSTSTORE_PASSWORD
+      EUROPEAN_IDENTITY_ENABLED: true
+      HUB_CONNECTOR_ENTITY_ID: ${HUB_CONNECTOR_ENTITY_ID}
+      TRUST_ANCHOR_URI: https://$SIGNIN_DOMAIN/SAML2/metadata/trust-anchor
+      METADATA_SOURCE_URI: https://$SIGNIN_DOMAIN/SAML2/metadata/aggregator
+      ENCRYPTION_KEY: $ENCRYPTION_KEY
+      ENCRYPTION_CERT: $ENCRYPTION_CERT
+      SIGNING_KEY: $SIGNING_KEY
+      SIGNING_CERT: $SIGNING_CERT
+      ENVIRONMENT: $APP_ENV
```

Comment:
we need to make sure that `$APP_ENV` and `$SIGNIN_DOMAIN` are set when template is instatiated - we saw issues today that they were not causing MSA breakage :(

Reply:
They are set when the template is instantiated: https://github.com/alphagov/verify-infrastructure-config/pull/179/commits/c30f1347866e05c1c4ed96ae1a5292fd59289e8a

Reply:
We got to the bottom of this - the changes to the pipeline weren't applied so the parameters weren't available yet. Now changes have been applied it looks good

Reply:
So I think what went wrong here is I had forgotten to set-pipeline after the above linked commit, this branch got deployed without that important change and this caused the staging problem 2 days ago.